asreview/base.py

```diff
@@ -216,6 +216,10 @@ def __init__(self,
         self.n_prior_excluded = n_prior_excluded
 
     def _prior_knowledge(self):
+        if self.n_prior_included is not None and self.n_prior_included < 0:
```

Comment:
I don't like the -1 option. If a -1 is given to the program, it should raise a ValueError (by design). At the moment, I don't see the actual problem.